.dockerignore

```diff
@@ -0,0 +1,2 @@
+.venv
+.env
```

Comment:
add new empty line 
also consider filling out this file in more detail 
for example` .idea/ .pytest_cache/`